src/main/java/com/razorpay/ApiClient.java

```diff
@@ -159,7 +166,7 @@ private void throwServerException(int statusCode, String responseBody) throws Ra
   private Class getClass(String entity) {
     try {
       String entityClass =
-          "com.razorpay." + Character.toUpperCase(entity.charAt(0)) + entity.substring(1);
+          "com.razorpay." + WordUtils.capitalize(entity, new char[]{'_'}).replaceAll("_", "");
```

Comment:
`WordUtils.capitalize(entity, '_')` will also work, as it takes variable length argument

Reply:
Second argument required is of type char[], whereas in this case, its char.